config.ini

```diff
@@ -1,6 +1,6 @@
 [site]
-#url=https://testing.1lims.com/
-url=https://automation.1lims.com/
+url=https://testing.1lims.com/
+#url=https://automation.1lims.com/
```

Comment:
we can run it on automation instance now

Reply:
Done